contrib/buildinfo/test/src/BuildInfoTests.scala

```diff
@@ -104,7 +103,17 @@ object BuildInfoTests extends TestSuite {
         val Right((result, evalCount)) = eval.apply(BuildInfo.run(runResult.toString))
         assert(
           os.exists(runResult),
-          os.read(runResult) == scalaVersionString)
+          os.read(runResult) == scalaVersionString
+        )
+      }
+
+      "generatedSources must be a folder" - workspaceTest(BuildInfo) { eval =>
```

Comment:
I fail to see, where this test checks, that the result is a folder.

Reply:
Ok, it was implicitly done by checking against the `generatedSourcesPath`, which is a folder. But that's true, it can be done better ;-) I added an explicit check.